src/Test.UnitTests.Sarif.Multitool.Library/TestData/MergeCommand/ExpectedOutputs/DuplicatedResults.sarif

```diff
@@ -1,11 +1,45 @@
 {
-  "version": "2.1.0",
   "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
```

Comment:
>version [](start = 3, length = 7)

Looks good! Please get some example PREfast files and let's test them explicitly.